### PR TITLE
Improve .netstandard 2.0 compatibility fix #2056

### LINKED ISSUE #2056: Some projects can't be consumed from .net 4.7.2
### Describe the bug
These projects below
Paramore.Brighter.MsSql.EntityFrameworkCore 
Paramore.Brighter.ServiceActivator.Extensions.HealthCheck 

Can't be consumed from .net472 because they don't target .netstandard2.0

### To Reproduce
For this specific case would be overkill to create a repo to reproduce the issue
I'm going to submit a Pull-Request to improve the compatibility

### Further technical details
https://github.com/BrighterCommand/Brighter/blob/master/src/Paramore.Brighter.MsSql.EntityFrameworkCore/Paramore.Brighter.MsSql.EntityFrameworkCore.csproj#L6
https://github.com/BrighterCommand/Brighter/blob/master/src/Paramore.Brighter.ServiceActivator.Extensions.HealthChecks/Paramore.Brighter.ServiceActivator.Extensions.HealthChecks.csproj#L4

### PR DESCRIPTION
**Pull request by IlSocio**

### CODE REVIEW

**Comment by DevJonny** [comment]
Thanks for the PR @IlSocio. Once you've signed the [Contributor License Agreement](https://cla-assistant.io/BrighterCommand/Brighter?pullRequest=2057) we can get this merged.

File: src/Paramore.Brighter.MsSql.EntityFrameworkCore/Paramore.Brighter.MsSql.EntityFrameworkCore.csproj [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
-    <PropertyGroup>
-      <Description>TThis is the MsSql Connection provider to obtain the connection from Entity Framework Core.</Description>
-      <Authors>Paul Reardon</Authors>
-      <TargetFrameworks>netstandard2.1;net6.0</TargetFrameworks>
-      <PackageTags>RabbitMQ;AMQP;Command;Event;Service Activator;Decoupled;Invocation;Messaging;Remote;Command Dispatcher;Command Processor;Request;Service;Task Queue;Work Queue;Retry;Circuit Breaker;Availability</PackageTags>
-    </PropertyGroup>
+  <PropertyGroup>
+    <Description>TThis is the MsSql Connection provider to obtain the connection from Entity Framework Core.</Description>
+    <Authors>Paul Reardon</Authors>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
```
Comment (iancooper) [inline src/Paramore.Brighter.MsSql.EntityFrameworkCore/Paramore.Brighter.MsSql.EntityFrameworkCore.csproj:6]: Do we need netstandard2.1? If we can be netstandard2.0 I would have thought that netstandard2.1 is not required. We could just choose netstandard2.0 and net6.0

**Review by iancooper — APPROVED** [review]
We can approve, but wondering if it is worth removing the nestandard2.1 referenc

File: src/Paramore.Brighter.MsSql.EntityFrameworkCore/Paramore.Brighter.MsSql.EntityFrameworkCore.csproj (hunk 1 above)
Comment (IlSocio) [reply src/Paramore.Brighter.MsSql.EntityFrameworkCore/Paramore.Brighter.MsSql.EntityFrameworkCore.csproj:6]: good point, totally agree... adding the commit.

**Review by iancooper — APPROVED** [review]
Thanks for that

**Comment by iancooper** [comment]
Thanks for contributing!!

**Pull request merged**